.github/workflows/release-github.yaml

```diff
@@ -13,8 +13,8 @@ jobs:
     if: github.event.pull_request.merged && startsWith(github.event.pull_request.head.ref, 'release-')
     runs-on: ubuntu-18.04
     needs:
-      - build
-      - cpp
+      - build-java
+      - build-cpp
```

Comment:
I renamed these jobs so that they're easier to understand when viewing the workflow as a whole